Update intercom.md (#132)
amplitude/amplitude-dev-center: docs/data/destinations/intercom.md

Under **Send Events**, make sure the toggle is enabled ("Events are sent to Intercom") if you want to stream events to Intercom. When enabled, events are automatically forwarded to Intercom when they're ingested in Amplitude. Events aren't sent on a schedule or on-demand using this integration. Events are sent to Intercom as [Intercom data events](https://developers.intercom.com/intercom-api-reference/reference/the-data-event-model). Intercom has a limit of 120 event types.

> Note that `[Intercom] event.created` is triggered whenever an event is created within Intercom, including Amplitude’s Event Streaming integration. 

Reviewer: ```suggestion
Intercom triggers an event `[Intercom] event.created` when Intercom creates events, including events from Amplitude's Event Streaming integration.
```

Author: Had too long of a break for Christmas.. I totally forgot about the use of passive voice haha

> If you don't want to store these events within Amplitude, you can use our [block or drop filters](https://help.amplitude.com/hc/en-us/articles/16805784778907-Remove-invalid-or-incorrect-data) to remove this data. 

Reviewer: ```suggestion
If you don't want Amplitude to store these events, use Intercom's [block or drop filters](https://help.amplitude.com/hc/en-us/articles/16805784778907-Remove-invalid-or-incorrect-data) to remove them.
```

Author: Block or drop filters are offered by Amplitude, so they're Amplitude's block or drop filters! 
```suggestion
If you don't want Amplitude to store these events, use Amplitude's [block or drop filters](https://help.amplitude.com/hc/en-us/articles/16805784778907-Remove-invalid-or-incorrect-data) to remove this data. 
```
